cpp/include/cudf/detail/stream_compaction.hpp

```diff
@@ -71,6 +71,7 @@ std::unique_ptr<table> drop_duplicates(
   std::vector<size_type> const& keys,
   duplicate_keep_option keep,
   null_equality nulls_equal           = null_equality::EQUAL,
+  null_order null_precedence          = null_order::BEFORE,
```

Comment:
I wish if there is a way to access these defaulted arugments.

Reply:
how do you mean? expose the argument to the non-detail API?

Reply:
I meant in C++.
We repeat `null_order::BEFORE` at several places.